[BUG] LCD_LANGUAGE el_gr fails to compile, bugfix-2.0.x

### Bug Description

Setting LCD_LANGUAGE el_gr fails to compile with the error

`Marlin/src/HAL/HAL_AVR/../../inc/../core/../lcd/language/language_el_gr.h:123:24: error: redefinition of 'const char Language_el_gr::MSG_VMAX_E []'`

there are two  "PROGMEM Language_Str MSG_VMAX_E"  lines in language_el_gr.h. this causes the error. The second MSG_VMAX_E should be MSG_VMAX_EN (trailing N) 

### My Configurations

[Configuration.zip](https://github.com/MarlinFirmware/Marlin/files/4080249/Configuration.zip)

### Steps to Reproduce

1. Set LCD_LANGUAGE el_gr in Configuration.h
2. Enable a GLCD eg REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
3. Attempt to compile

**Expected behavior:**
It should compile

**Actual behavior:**
It errors

--- a/Marlin/src/lcd/language/language_el_gr.h
+++ b/Marlin/src/lcd/language/language_el_gr.h
@@ -120,7 +120,7 @@ namespace Language_el_gr {
   PROGMEM Language_Str MSG_VMAX_B                          = _UxGT("Vμεγ ") LCD_STR_B;
   PROGMEM Language_Str MSG_VMAX_C                          = _UxGT("Vμεγ ") LCD_STR_C;
   PROGMEM Language_Str MSG_VMAX_E                          = _UxGT("Vμεγ ") LCD_STR_E;
-  PROGMEM Language_Str MSG_VMAX_E                          = _UxGT("Vμεγ *");
+  PROGMEM Language_Str MSG_VMAX_EN                         = _UxGT("Vμεγ *");
   PROGMEM Language_Str MSG_VMIN                            = _UxGT("Vελαχ");
   PROGMEM Language_Str MSG_VTRAV_MIN                       = _UxGT("Vελάχ. μετατόπιση");
   PROGMEM Language_Str MSG_ACCELERATION                    = _UxGT("Accel");

